### PR TITLE
Glossary update Aug 18

### PR DESCRIPTION
**Pull request by hazelshapiro**
Deleting redundant/outdated section, and adding a few additional updates/clarifications throughout.

### CODE REVIEW

File: glossary.md [hunk 1]
```diff
@@ -81,23 +82,33 @@ rating(s) and other fields associated with a `response`. The following specifica
...
 
 * `Link`: Defines which `survey objects` are connected. The rating is applied to that `link`.
-* `Performance rating`: 0-100 rating of the performance of the subject. Answers the question: 
-  What is your satisfaction with this input? (0=No performance, 100 = perfect)
 * `Criticality rating`: 0-10 rating of the criticality of an input to an output,
   e.g. criticality of an `observing system` to a `data product`. This answers the question:
   On a scale of 1-10, how much would the loss of this input impact the performance of your 
   `data product` or `application` (1 - very little impact; 10 - complete loss of performance).
-* `Rationale`: Why a `criticality` or `performance` rating was selected.
+* `Criticality rationale`: Text description answering the question: What accounts for this
```
Comment (rmarow) [inline glossary.md:89]: @hazelshapiro looks like there will now be 2 bullets for criticality rationale? did you just want to add this info to the original bullet or replace?

File: glossary.md [hunk 2]
```diff
@@ -81,23 +82,33 @@ rating(s) and other fields associated with a `response`. The following specifica
...
+The `node color` of an `observing system` or a `data product` is defined in this 
 document: https://docs.google.com/presentation/d/1RmEGcPkC3_9o3qeAndv0QvAcdZwFIHC-/edit#slide=id.g1e651286dde_0_54
 The `node color` of an `application` is rated separately based on the application performance
-compared to the `application performance criteria`.
+compared to the `application performance criteria`. 
+
+While only a  small number of respondents will provide responses on `Observing Systems` through 
+`Applications`, a larger number of respondents (up to ~20)  will evaluate the relationships between 
+`Applications` and `Societal Benefit Areas`. For now, we expect users to follow instructions, but
+eventually, we may want to apply restrictions so that the right people can only fill out the right
```
Comment (rmarow) [inline glossary.md:110]: Great callout

**Review by rmarow — CHANGES_REQUESTED** [review]

File: glossary.md (hunk 1 above)
Comment (MattF-NSIDC) [reply glossary.md:89]: Maybe I'm missing the duplicate, but not seeing it. We have `Criticality rating` and `Criticality rationale` as separate fields that look really similar though :)

Comment (MattF-NSIDC) [reply glossary.md:89]: This addresses this issue: https://github.com/nsidc/usaon-vta-docs/issues/9 @hazelshapiro would you agree that this PR resolves that issue fully?

Comment (rmarow) [reply glossary.md:89]: @MattF-NSIDC my eyes really need checked today!

**Review by rmarow — APPROVED** [review]

**Comment by rmarow** [comment]
@hazelshapiro going ahead and merging this !

File: glossary.md (hunk 1 above)
Comment (hazelshapiro) [reply glossary.md:89]: Yes, I think so! It's hard with so many similar fields. There should be criticality rating / criticality rationale / performance rating / performance rationale.

**Pull request merged**